### PR TITLE
Change sort UI in MyMangaReader CMS source

### PR DESCRIPTION
**Pull request by null-dev**
Also removed dead sources, re-enabled alive sources and updated source genres/tags.

New list of sources included in this extension:
- AR: مانجا اون لاين
- AR: Manga FYI
- EN: Read Comics Online
- EN: Fallen Angels Scans
- EN: Mangawww Reader
- EN: Manga Spoil
- EN: Manga Forest
- EN: Chibi Manga Reader
- EN: ZXComic
- EN: Mangacox
- EN: Hentai2Manga
- EN: White Cloud Pavilion
- EN: 4 Manga
- EN: XYXX.INFO
- ES: My-mangas.com
- ES: SOS Scanlation
- ES: Doujin Hentai
- FR: Manga-LEL
- FR: Manga Etonnia
- FR: Scan FR
- FR: Manga FYI
- FR: Mugiwara
- FR: scans-manga
- FR: Henka no Kaze
- ID: MangaOnline
- ID: Komikid
- ID: MangaID
- ID: Manga Seru
- ID: Manga FYI
- ID: Bacamangaku
- JA: Mangazuki RAWS
- JA: マンガ/漫画 マガジン/雑誌 raw
- PL: ToraScans
- PT: Comic Space
- PT: Mangás Yuri
- RU: NAKAMA
- TR: MangaHanta
- TR: ManhuaTR
- VI: Fallen Angels Scans
- OTHER: HentaiShark

### CODE REVIEW

**Comment by CarlosEsco** [comment]
Update the build.gradle versions

File: src/all/mmrcms/genSources.sh [hunk 1]
```diff
@@ -288,22 +291,24 @@ gen "id" "Manga FYI" "http://mangafyi.com/manga/indonesian"
...
-gen "ja" "MangaRAW" "https://www.mgraw.com"
+# Went offline
+# gen "ja" "MangaRAW" "https://www.mgraw.com"
 gen "ja" "マンガ/漫画 マガジン/雑誌 raw" "http://netabare-manga-raw.com"
 gen "pl" "ToraScans" "http://torascans.pl"
 gen "pt" "Comic Space" "https://www.comicspace.com.br"
 gen "pt" "Mangás Yuri" "https://mangasyuri.net"
 gen "ru" "NAKAMA" "http://nakama.ru"
 # Went offline
 # gen "ru" "AkaiYuhiMun team" "https://akaiyuhimun.ru/reader"
```
Comment (Taumer) [inline src/all/mmrcms/genSources.sh]: Moved to `https://akaiyuhimun.ru/manga/`

**Comment by wand09** [comment]
please add

mangashiro.net
mangaindo.web.id
mangacanblog.com

for Indonesian list

**Comment by wand09** [comment]
and i can't load / download images for all manga on site MangaID (Indonesia) extension (My manga Reader CMS) application version 0.7 3 in android 7.1

please fix this problem

**Comment by null-dev** [comment]
@CarlosEsco Whoops, nice catch, fixed!

@wand09 mangashiro.net, mangaindo.web.id and mangacanblog.com are not MyMangaReader sources so I can't add them here. I've fixed MangaID though.

**Comment by wand09** [comment]
images on manga still can not be loaded, and just stop in downloading text

**Comment by null-dev** [comment]
@wand09 This change hadn't been rolled out yet. Wait until this pull request is merged.

**Pull request merged**